src/ui/components/RecipeEnrollmentPage.tsx

```diff
@@ -53,6 +144,21 @@ const RecipeEnrollmentPage: FC = () => {
           Enroll
         </Button>
       </Form>
+
+      <Modal show={!!enrollError} onHide={handleModalClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Force Enrollment</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>{EnrollmentError(slug, enrollError)}</Modal.Body>
```

Comment:
EnrollmentError is a component:

```suggestion
        <Modal.Body><EnrollmentError slug{slug} enrollError={enrollError} /></Modal.Body>
```